plasmapy/formulary/braginskii.py

```diff
@@ -502,9 +515,21 @@ def thermoelectric_conductivity(self):
     @property
     @validate_quantities
     def ion_thermal_conductivity(self) -> u.W / u.m / u.K:
-        """
+        r"""
         Calculate the thermal conductivity for ions.
 
+        The ion thermal conductivity (:math:`\kappa`) of a plasma is defined by
```

Comment:
Just a thought - one key piece of intuition I was missing while reviewing these three years ago was the understanding that the way these are *defined* is a transport law like $\vec{q} = -\kappa * \nabla T$, q being the heat flux caused by the temperature gradient. What transport theory actually does is provide estimates, based on reasonable assumptions, for the magnitude of $\kappa$ and other such coefficients. And then you can have different variants, including curvature effects (thus, neoclassical), or turbulence (thus, anomalous).

So yeah, despite wondering a little about the word "defined", I think this is a good change, but I wonder whether we shouldn't rewrite the docs for the whole transport module a little, to account for that...

gah, of course we should do that, we should take a hammer to the whole thing, after all. I'll make a note to do that when we refactor it.

Reply:
But at the same time, those transport laws are very general and don't necessarily apply only to classical transport. So it might be a moot point.

Reply:
Interesting points. We could change "defined" if users or contributors might be confused by this term. Perhaps something like "... is related by the formula ..." or "This function computes ... by the formula ...". But I think that we should discuss this and figure this out in an Issue or Discussion so that the documentation can be standardized.

Reply:
Good point, let's come back here from #1044 later :)